catroid/src/org/catrobat/catroid/common/SoundInfo.java

```diff
@@ -76,23 +78,11 @@ public SoundInfo copySoundInfoForSprite(Sprite sprite) {
 
 	public String getAbsolutePath() {
 		if (fileName != null) {
-			return Utils.buildPath(getPathToSoundDirectory(), fileName);
-		} else {
-			return null;
-		}
-	}
-
-	public String getAbsolutePathBackPack() {
-		if (fileName != null) {
-			return Utils.buildPath(getPathToBackPackDirectory(), fileName);
-		} else {
-			return null;
-		}
-	}
-
-	public String getAbsolutePathBackPackSound() {
-		if (fileName != null) {
-			return Utils.buildPath(getPathToBackPackSoundDirectory(), fileName);
+			if (isBackpackSoundInfo) {
+				return Utils.buildPath(getPathToBackPackSoundDirectory(), fileName);
+			} else{
```

Comment:
Looks like we do not have this rule in our ruleset, because `Catroid-Build-SourceTests-StaticChecks` was green. (@robertpainsi to the rescue!) :cactus: